src/ceph-volume/ceph_volume/devices/raw/list.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, argv):
     def generate(self, devs=None):
         logger.debug('Listing block devices via lsblk...')
         info_devices = disk.lsblk_all(abspath=True)
-        if devs is None or devs == []:
+        if devs is None or devs == [] or devs == [None]:
```

Comment:
would `if not devs or not any(devs)` be a better way to put this? For example
```
class PlaceHolder:
    def __init__(self):
        self.x = 'x'

x1 = None
x2 = []
x3 = [None]
x4 = [2]
x5 = [PlaceHolder()]

for x in [x1, x2, x3, x4, x5]:
    if not x or not any(x):
        print('did not pass')
    else:
        print('passed')
```
would give you
```
did not pass
did not pass
did not pass
passed
passed
```